make/modules/jdk.hotspot.agent/Lib.gmk

```diff
@@ -31,7 +31,7 @@ ifeq ($(call isTargetOs, linux), true)
   SA_CFLAGS := -D_FILE_OFFSET_BITS=64
 
 else ifeq ($(call isTargetOs, macosx), true)
-  SA_CFLAGS := -Damd64 -D_GNU_SOURCE -mno-omit-leaf-frame-pointer \
```

Comment:
Is this really proper for macos-x64? I thought we used -Damd64 as a marker for all macos-x64 C/C++ files. (Most files get their flags from the common setup in configure, but SA is a different beast due to historical reasons).

Reply:
@luhenry , could you please check this comment, I think SA-agent was MS's job here.

Reply:
The target is identified by the header file now:
https://github.com/openjdk/jdk/pull/2200/files#diff-51442e74eeef2163f0f0643df0ae995083f666367e25fba2b527a9a5bc8743a6R35-R41

Do you think there is any problem with this approach?

Reply:
@lewurm No, that's okay. I just wanted to know that this had not been lost.